src/components/dialogs/load-creation-dialog.js

```diff
@@ -0,0 +1,483 @@
+
+const filter = createFilterOptions();
+
+const useStyles = makeStyles((theme) => ({
+    helperText: {
+        margin: 0,
+    },
+    popper: {
+        style: {
+            width: 'fit-content',
+        },
+    },
+}));
+
+/**
+ * Dialog to create a load in the network
+ * @param {Boolean} open Is the dialog open ?
+ * @param {EventListener} onClose Event to close the dialog
+ * @param {String} title Title of the dialog
+ */
+const LoadCreationDialog = ({ open, onClose, network }) => {
+    const classes = useStyles();
+
+    const studyUuid = decodeURIComponent(useParams().studyUuid);
+
+    const intl = useIntl();
+    const intlRef = useIntlRef();
+
+    const { enqueueSnackbar } = useSnackbar();
+
+    const [busOrBusbarSectionOptions, setBusOrBusbarSectionOptions] = useState(
+        []
+    );
+
+    const [loadId, setLoadId] = useState('');
+
+    const [loadName, setLoadName] = useState('');
+
+    const [loadType, setLoadType] = useState('');
+
+    const [activePower, setActivePower] = useState('');
+
+    const [reactivePower, setReactivePower] = useState('');
+
+    const [voltageLevel, setVoltageLevel] = useState('');
+
+    const [busOrBusbarSection, setBusOrBusbarSection] = useState('');
+
+    const [errors, setErrors] = useState(new Map());
+
+    const handleChangeLoadId = (event) => {
+        setLoadId(event.target.value);
+    };
+
+    const handleChangeLoadName = (event) => {
+        setLoadName(event.target.value);
+    };
+
+    const handleChangeLoadType = (event) => {
+        setLoadType(event.target.value);
+    };
+
+    const handleChangeActivePower = (event) => {
+        // Replace ',' by '.' to ensure double values can be parsed correctly
+        setActivePower(event.target.value?.replace(',', '.'));
+    };
+
+    const handleChangeReactivePower = (event) => {
+        // Replace ',' by '.' to ensure double values can be parsed correctly
+        setReactivePower(event.target.value?.replace(',', '.'));
+    };
+
+    const handleChangeVoltageLevel = (event, value, reason) => {
+        if (reason === 'select-option') {
+            setVoltageLevel(value);
+            setBusOrBusbarSection('');
+            if (value?.topologyKind === 'NODE_BREAKER') {
+                // TODO specify the correct network variant num
+                fetchBusbarSectionsForVoltageLevel(studyUuid, 0, value.id).then(
+                    (busbarSections) => {
+                        setBusOrBusbarSectionOptions(busbarSections);
+                    }
+                );
+            } else if (value?.topologyKind === 'BUS_BREAKER') {
+                // TODO specify the correct network variant num
+                fetchBusesForVoltageLevel(studyUuid, 0, value.id).then(
+                    (buses) => setBusOrBusbarSectionOptions(buses)
+                );
+            } else {
+                setBusOrBusbarSectionOptions([]);
+            }
+        } else if (reason === 'clear') {
+            setVoltageLevel(null);
```

Comment:
Is the initlal or default value of 'voltageLevel' state equal to null or '' ?
In clearValues function, it is set to ''